### ISSUE TITLE
Panzer: modify DOF Manager for DG support

### ISSUE THREAD

**Issue by rppawlo** [trilinos/Trilinos#678]
@trilinos/panzer

**Comment by kyungjoo-kim** [comment]
Hi Roger,


I and @eric-c-cyr  made some changes in the DOF managers for high order elements. The changes will make the the dof manager more generic and follow the basic assumption (previously the geometric field pattern is written on the assumption that the max intrepid order is 2 and DOFs are aligned geoemtrically which is true for C2 elements but not true fro Cn ( n = 2 ) ).

I am wondering how we can work this without many conflicts as it may involve fundamental design changes.



Best,
Kyungjoo

**Comment by rppawlo** [comment]
We have a meeting 8am MT Wednesday morning to discuss DG design issues in Panzer. Any chance you could attend?

**Comment by kyungjoo-kim** [comment]
Yes, I can attend but Eric already knows all modifications that we made in Panzer. I just want to push my commits. See you then.

**Comment by rppawlo** [comment]
ok. We have not started the DOF work for DG yet so it is probably safe to push. Did you run all the drekar tests? If you would like to me to do that before you push, let me know where to pull the changes from to test.

**Comment by kyungjoo-kim** [comment]
I ran panzer unit tests and asked eric to run the drekar tests. The modification is not invasive but I would like to make sure that I do not break panzer.

**Comment by eric-c-cyr** [comment]
I will run these tests shortly. As soon as I can work for a few minutes :(

**Comment by rppawlo** [comment]
code was reviewed and committed. Closing.
